loadtests/lambdas/variables.tf

```diff
@@ -162,43 +162,38 @@ variable "keycloak_v2_prod_url" {
   default     = ""
 }
 
-variable "keycloak_v2_dev_client_id" {
+variable "keycloak_v2_dev_username" {
   type        = string
-  description = "Keycloak v2 service account client id in customer dev environment"
-  default     = "terraform-cli"
+  description = "The username of the user used by the provider for authentication via the password grant"
+  default     = ""
 }
 
-variable "keycloak_v2_test_client_id" {
+variable "keycloak_v2_dev_password" {
```

Comment:
I would add sensitive = true on these password ones to make sure they won't print out in any github actions accidently